Islandora-CLAW/CLAW
# Install D8 and modules using composer

Install PR for #474 and #475 

### Testing instructions
* Pull down the code from this PR
```
git checkout -b dannylamb-composer master
git pull https://github.com/dannylamb/CLAW.git composer
```
* visit https://github.com/settings/tokens and get an oauth token (check the 'repos' checkbox when making it)
* put your token in `install/configs/variables` as the `GITHUB_TOKEN` variable (it's set to 'changeme' by default)
* vagrant up

Functionality should remain the same.  I tested by creating, editing, and then deleting a fedora resource, and made sure the rules actions were firing.
